Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -95,6 +95,18 @@ public void SerializesAndDeserializesCustomPropertyNames()
         {
             Check(TestObjects.fmRadio);
         }
+        
+        [TestMethod]
+        public void SerializesAndDeserializesObjectsWithIonConstructor()
```

Comment:
We should be testing with a non-default value.

Reply:
Addressed.